ui/scripts/domains.js

```diff
@@ -69,11 +69,23 @@
                                 async: false,
                                 success: function(json) {
                                     var jid = json.deletedomainresponse.jobid;
+                                    var wasDomainDeletedWithSuccess = false;
+                                    $.ajax({
+                                        url: createURL("queryAsyncJobResult&jobId=" + jid),
+                                        dataType: "json",
+                                        async: false,
```

Comment:
async `false` will block the UI... is there a way to have this but without blocking the UI?

Reply:
that is a good question. If you take a look, the upper method "deleteDomain" (at line 69), it is already an "async" call. I could for instance, move the code from lines 81-93 to the success function (line 78), but that can cause an odd behavior if the user clicks somewhere in the UI before the method finishing. That is why I decided to roll with the "standard" and use a "sync" call as well.

Reply:
@rhtyd do you think this can be a problem?

Reply:
@rafaelweingartner sorry for a late reply, you don't need to query job results manually but use `args.response.success` and pass the `jobId`. Ideally an async notification callback could have been explored and used.

Reply:
That is actually what is being used right now. I tried though to use the " args.response.success" with a custom function to render the HTML in case of success. However, I was not able to find a way to send a function there. The only way I have seen people using is the following:

```
                                    args.response.success({
                                        _custom: {
                                            jobId: jid
                                        }
                                    });
```
Which takes only the jobid into account, and does not provide me a callback. 

Would you like me to revert this one then?